doc/source/whatsnew/v1.1.0.rst

```diff
@@ -686,6 +686,7 @@ Numeric
 - Bug in DataFrame reductions using ``numeric_only=True`` and ExtensionArrays (:issue:`33256`).
 - Bug in :meth:`DataFrame.corr` and :meth:`DataFrame.cov` raising when handling nullable integer columns with ``pandas.NA`` (:issue:`33803`)
 - Bug in :class:`DataFrame` and :class:`Series` addition and subtraction between object-dtype objects and ``datetime64`` dtype objects (:issue:`33824`)
+- Bug in :class:`DataFrame` flex arithmetic methods with ExtensionDtypes (:issue:`????`)
```

Comment:
You can remove this, it's not a bug fix, only a regression on master

Reply:
updated